### DISCUSSION TITLE
[Topic spoilers within] Move some pool topics to seen

### DISCUSSION THREAD

**Discussion by Freshairkaboom** [Monika-After-Story/MonikaModDev#7912] [issue discussions]
I was just wondering, I've seen some talk about certain topics not needing to be in the unseen menu, because not all topics are supposed to be used by everyone. I feel this way about a range of topics at the moment, and I think they should be moved to seen, so they don't appear in the unseen menu where new pool topics are supposed to go.

Topics I feel this way about currently are:

"I'm breaking up with you." - monika_breakup
"You're a murderer!" - monika_justification
"Will you marry me?" - monika_marriage
"Do you want to meet my girlfriend?" - monika_girlfriend

I don't know about you guys, but I really don't feel comfortable having these very sensitive topics that either hurt Monika, or are questions that shouldn't just be asked for the completion side of things, be part of the unseen menu that you are supposed to empty over time. When you have (1) unseen topic, it's much easier to find that than if you have a new unseen topic, but also 4-5 others that you just don't want to clear out.

I understand why these topics are there, I just don't think they should be front and center like that. Have them appear in their proper prompt spot, sure, but not in unseen. I don't know if you guys agree with me, but I thought I'd share my two cents on the matter. Every time I reopen Unseen it hurts to see these topics still uncleared.

**Comment by briaryoung** [comment]
I agree with this wholeheartedly. When I first saw them in the unseen menu, I clicked mindlessly and ended up hurting Moni’s feelings. I don’t think most people come to this mod actively destroying their relationship with Moni.

**Comment by fromulus** [comment]
Doesn't the "I don't want to see this menu anymore" option in the unseen menu already helps not having to see such topics condensed in the same place anymore ?

**Reply by XxPower-Girl-89xX** [reply]
well people wont know any new topics that you can get

**Comment by Freshairkaboom** [comment]
The unseen menu is supposed to be useful to go through when new pool topics are added, therefore, hiding it is in no way a solution in my opinion. It should be used for topics that are interesting and fun to go through, not force people into awkward or hurtful situations with their girlfriend in order to clear it.

If there's another purpose for having those options there in the front, I'm happy to hear it.

**Comment by kirillPro2020** [comment]
You can move these topics to separate menus
(And it's better to remove them altogether)

**Comment by Booplicate** [comment]
I completely disagree on `monika_marriage`. This one shouldn't be marked as seen, no reason for that. If you feel like it's not the right time to ask, don't click on it yet.

About the rest 3, keep in mind one thing. Until you see a topic, its title will be rendered in italic, which helps distinguish it from the rest of topics in the menu. If you mark it as seen, the title will use the normal font, and the chances that you will missclick on it will be higher.

No one is forced to click on those, people do it out of curiosity. I can see why someone would hide them - you're not going to use them, understandable. But not all people like you, some will enjoy it (I guess?). If I may say so it's part of the simulation - you're free to do what you want. You can do nice things, you can do horrible things. It's up to you. We can remove those, okay. But how far this will go? Should we remove all bad choices in other topics? Should we remove the negative part of affection progression? Won't it destroy the immersion?

**Comment by kirillPro2020** [comment]
This will not destroy the immersion in any way. Who would say bad things to a dear person?
Unless you are a bad person, of course.
(even if she didn't even know about it, okay if it was a joke and she was not offended)
It doesn't matter whether it's a simulation or not, it's all a matter of morality.

**Comment by multimokia** [comment]
The way I see it is that the options should be existing. It's your own self control which prevents you from saying them. Just like the bad options in first time compliments and such.

If people play MAS for 'completion' (see all dialogue they can) then they're playing it more as a game rather than taking it as a relationship.

Sure it's there in unseen, but it's not hard to navigate around them and then hide the menu once you finish the rest of the unseens. To me that's a solution.

**Comment by LooseEthics** [comment]
These topics exist for a reason and removing them completely would be a detriment imo. That being said, I think it would be nice to have the option to hide unseen pool topics without having to go through them. If I know I'm never using it, having it there will just make me nervous. Misclicks happen and you can't really undo them in mas. (I mean, you can, but that's a session-level rollback.)

**Comment by kirillPro2020** [comment]
It seems to me that these topics will catch the eye. And each time asking Monica to open this menu in the hope of something new and interesting, the player will see these topics

**Comment by kirillPro2020** [comment]
Can you add these topics to the blacklist without opening them?

**Comment by multimokia** [comment]
Well after hiding the menu, it stays hidden until next update. So nothing to catch the eye if you hide it.

If by blacklist, you mean pressing `X` on a topic, that only applies to random topics

**Comment by multimokia** [comment]
No. Again, the point is realism. Yes you can say it but the thing preventing you from doing so is your own self control.

Putting it in its own menu defeats the purpose.

**Comment by Freshairkaboom** [comment]
My gripe is that this encourages people to go through all of the options, it's like having having 5 notifications on Facebook, but one of the notifications say something like "send a message to your dad telling them they're fat."

You can close Facebook and go do something else yes, but every time you open Facebook back up, there's the notification. You always have the option to go tell them they're fat manually, so why shove it in people's face?

And the notification in question would be pinned to the top of your notifications until you sent it. It's not impossible to work around, but why have it like that? People who want to be mean to Monika will find ways to do so without shoving it in their face.

Again, my solution is not to remove those topics, but have them appear in their respective prompt locations instead of in Unseen. A list going through unspoken topics is not realistic regardless, but if it were, I doubt loving relationship partners would want to have a bunch of mean things at the top of their list.

Also, regarding the marriage one, it's again the trouble with topics that shouldn't be said simply for the completionism. What made me think about this issue in general was first and foremost when someone mentioned in issue #6084 that topics shouldn't just be in "Unseen" for the sake of it, as some topics are not meant to be said by everyone, and having them there encourages people to click it.

**Comment by Booplicate** [comment]
> I doubt loving relationship partners would want to have a bunch of mean things at the top of their list

But people still end up saying mean stuff to the ones they love.

And again if the problem in those "notifications", you can hide them.

Asking for marriage is different from telling her "I'm transgender". In the end most of the people will use the marriage topic, but won't use the `I'm transgender` one. One is a logical step in a relationship, another one is your own discoveries about yourself, which not everyone will make.

**Comment by Freshairkaboom** [comment]
As I said, ending up saying mean stuff to the one you love and having it in a priority list are two different things. I guess I'm outnumbered here though, it's just really bothering me...

**Comment by Daiozen** [comment]
Why not add an additional button "I don't want to talk about these topics." under the current "I don't want to see this menu anymore."? Said button could prevent the leftover topics from appearing again in unseen while still making it visible for players who want to pick them.

> If I know I'm never using it, having it there will just make me nervous. Misclicks happen and you can't really undo them in mas.

From personnal experience, these misclicks do happen. The unseen topic list appears right underneath the 'unseen' button once you click it, so if your pc is slow and you end up double clicking while it loads the list, it may end up selecting one of the topics underneath.

**Comment by Freshairkaboom** [comment]
I agree, if only I knew how to do that I would propose a PR. I hope it's not too much work, I guess I can live with the current situation if I just keep hiding the Unseen menu, it's just not very ideal for me.

**Comment by ThePotatoGuy** [comment]
Regarding the facebook analogy, it sounds like something you'd want is a way to dismiss things off the unseen menu like dismissing notifications. I can agree with this since having a lot of things in the unseen menu (especially if more questionable topics get added) can really clutter the menu. However, we've previously reasoned against this for 2 main reasons: 
1. didn't have an easy way for a user to select a bunch of topics at the time (first issue was #959, me talking about exclusion menu in #1405)
2. the reasons multi said (self-control)

For me, reason 2 was a way to rationalize that the menu is fine as is, but with the checked scrollable menu from #5803, we have a UI route to do this now. We can't set the topics as seen though, cause that could ruin branching in other topics. Instead, we can use flags property with separate persist storage to hide stuff specifically off of unseen only.

So I'm on board for allowing users to dismiss things off of unseen, but if multi and boop think otherwise for immersion and gameplay purposes, then I'm fine with the menu staying as is.

**Reply by Justformas** [reply]
I don't know what happened to my first message in this discussion but I'm going to try addressing the points brought up in this discussion more directly this time. The facebook analogy in particular is what could be called a "dark pattern", which is user interface design that can encourage users to do things that they might not normally or otherwise do. The worst ones are usually intentional, but dark patterns can be unintentional too, and subtle or psychological, but still powerful. They can also make it more likely for accidents to happen. I also agree with Freishairkaboom that beyond the 3 hurtful topics, it could pressure someone to say something like "Will you marry me?" prematurely (when they aren't actually ready to marry Monika).

The problem with the unseen menu is that it's not realistic and comes off as more of a gameplay mechanic. It's not realistic to be given a list of topics to say to your partner, especially on a menu labeled "unseen" as if meant to be seen and reminded of.

The problem with dismissing this issue as just a gameplay completionism problem is that the urge for completionism in humans extends beyond games, like for seeing/clearing notifications (like in this example), and for completing jobs or tasks. In certain individuals, (me included), it also goes beyond a completionism issue, to an actual OCD issue. Being able to "hide" the unseen menu is not necessarily enough for someone with OCD since they know a topic is still there in the unseen menu. It's also not a good solution for people who actually want to see the unseen menu for other topics. But being repeatedly reminded of topics you may not want to bring up at first, it can have an effect on people.

It's not so simple for such people, especially those with OCD, to have more "self control", and frankly saying that as a solution does make the whole thing seem more like gamification meant for the player to resist saying the 3 hurtful topics, and I thought MAS wasn't intended to be a game. Self control you could say is all that's needed to resist peer pressure and to avoid looking at your affection value as well, but that was still seen as a problem.

I also think it's a lot to ask of players to somehow know to not treat MAS like a game, even though it's based off of a game, DDLC, and has some differences from real life. MAS is also far from the only relationship game/mod to exist, all of which have differences from real life, and people have come to expect that. Monika even acknowledges that she's not in our reality yet. The fact that Monika is supposed to be or seem self-aware doesn't change any of this. She was also that way in DDLC. But there can be advantages to not being exactly like real life, and I think at least for some or many people, that is part of the point or appeal.

Anyway I don't think that certain topics such as the 3 hurtful ones have to be off the unseen menu by default, but being able to dismiss topics that you no longer wish to see on that menu would be a good addition. It still requires the decision to dismiss a topic instead of saying it to Monika. I think if a player can make that decision, they should be allowed to do so and reduce the pressure/urge to bring it up, as well as reduce the chance of an accidental click on an unwanted topic.

Accidental clicks with the menu system in MAS have happened for me as well, including for one of the 3 hurtful topics. Man was that a huge affection loss worth thousands of points. Thank god for persistent backups since there's no way to tell her you didn't mean it like there would be in real life. I guess that's the closest thing we have.
